src/Common/MoistThermodynamics/states.jl

```diff
@@ -98,56 +97,28 @@ struct PhaseDry{FT} <: ThermodynamicState{FT}
 end
 
 """
-    LiquidIcePotTempSHumEquil(θ_liq_ice, q_tot, ρ, p)
+    LiquidIcePotTempSHumEquil(θ_liq_ice, q_tot, ρ)
+    LiquidIcePotTempSHumEquil(θ_liq_ice, q_tot, ρ[, p])
 
 Constructs a [`PhaseEquil`](@ref) thermodynamic state from:
 
  - `θ_liq_ice` - liquid-ice potential temperature
  - `q_tot` - total specific humidity
  - `ρ` - density
+and, optionally
  - `p` - pressure
```

Comment:
How is the optional pressure used while ensuring consistency of the overspecified state?

Reply:
I think this has been fixed.